README.md

```diff
@@ -47,6 +47,8 @@ Failing to follow the community guidelines as described in this document carries
 
 #### Privacy
 - All conversations are private unless otherwise specified. 
+  - The lone exception is in the event of seeking mediation or reporting behavior in volation of the code of conduct. In this case conversations may be shared with a member of the administrative team who may in turn share it with the rest of the administrative team if discussion is required.
```

Comment:
I find the word "lone" to be too fancy for this language. I recommend "only". And maybe I would phrase it "The only exception is when seeking mediation..." It comes across a bit more direct and to the point IMHO.

Philosophically I think it's a good idea to set the expectation that private messages are only private until and unless another participant feels they violate the code of conduct. Do you think it's important to spell out that, in the worst case, it might also be appropriate and necessary to then share the conversation with entities outside of the Slack? For example if a criminal threat were, god forbid, to be made in a private message, it should be permissable for an administrator to share that conversation with police. Is this something that needs to be covered explicitly?